.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,22 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/typescript-node
+{
+	"name": "Node.js & TypeScript",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "mcr.microsoft.com/devcontainers/typescript-node:1-20-bullseye"
+
+	// Features to add to the dev container. More info: https://containers.dev/features.
+	// "features": {},
```

Comment:
I recommend adding in the following to feature to get azure cli also as part of the dev container automatically.
```
        "features": {
		"ghcr.io/devcontainers/features/azure-cli:1": {
			"version": "latest"
		}
	}
```

Reply:
Do we need it?

Reply:
I'll add this in a separate PR